sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/TelemetryPartA.cs

```diff
@@ -122,5 +123,10 @@ internal static void InitRoleInfo(Resource resource)
                 RoleName = serviceName;
             }
         }
+
+        internal static string FormatTimestamp(System.DateTime timeStamp)
```

Comment:
This must be an UTC timestamp, otherwise the result is wrong.

```suggestion
        internal static string FormatUtcTimestamp(System.DateTime utcTimeStamp)
```

```csharp
    var localNow = DateTime.Now;
	var univNow = localNow.ToUniversalTime();
	var localNowStr = localNow.ToString("yyyy-MM-ddTHH:mm:ss.fffffffZ");
	var univNowStr = univNow.ToString("yyyy-MM-ddTHH:mm:ss.fffffffZ");
	Console.WriteLine($"Local: {localNowStr}");
	Console.WriteLine($"UTC: {univNowStr}");
```

Output:

```text
Local: 2021-06-18T13:21:19.3646975Z
UTC: 2021-06-18T20:21:19.3646975Z
```